CMakeLists.txt

```diff
@@ -93,14 +87,6 @@ list(APPEND EXTRA_LIBRARIES ${CMAKE_DL_LIBS})
 
 find_package(Boost 1.69.0 REQUIRED COMPONENTS mpi serialization system filesystem python${PYTHON_VERSION_NO_DOT} numpy${PYTHON_VERSION_NO_DOT} unit_test_framework)
 
-set (TEST_ENV "PYTHONPATH=${CMAKE_BINARY_DIR}:$ENV{PYTHONPATH}")
-
-if (APPLE)
```

Comment:
@XzzX just FYI, `RT_LIBRARIES` was used nowhere.

Reply:
I recall it might have been needed under OSX at some point.

Reply:
Ok, do we officially support OSX and/or Windows? If so we need runners...